grouper/fe/handlers/permissions_grant_tag.py

```diff
@@ -0,0 +1,73 @@
+from sqlalchemy.exc import IntegrityError
+from grouper.fe.forms import PermissionGrantTagForm
+from grouper.fe.util import GrouperHandler
+from grouper.models.audit_log import AuditLog
+from grouper.permissions import grant_permission_to_tag
+from grouper.models.permission import Permission
+from grouper.constants import TAG_EDIT
```

Comment:
nit: alphabetical sort imports please, probably should get flake8 to check this ([extra credit](https://github.com/public/flake8-import-order))